src/entities/communitySbt.ts

```diff
@@ -25,10 +28,37 @@ type MultiRedeemData = {
     roots: Array<Bytes>;
 }
 
+enum TxBadgeStatus {
+    SUCCESSFUL,
+    FAILED,
+    PENDING
+}
+
+enum BadgeClaimingStatus {
+    CLAIMED,
+    CLAIMING,
+    NOT_CLAIMED
+}
+
+export type BadgeWithStatus = {
+    badgeType: number;
+    claimingStatus: BadgeClaimingStatus;
+}
+
+export type GetBadgesStatusArgs = {
+    apiKey: string;
+    subgraphUrl: string;
+    season: number;
+    potenetialClaimingBadges: Array<number>;
```

Comment:
typo: potenetialClaimingBadges
also can we rename it to potentialClaimingBadgesTypes

Reply:
replaced in 2af91f07da29e87dba70e9893c14fcaf6c3d0c20